WTWSMS/history/characters/greek.txt

```diff
@@ -3625,4 +3625,32 @@
 	434.1.1={
 		death=yes
 	}
-}
\ No newline at end of file
+}
+1310010= {
+	name="Alexander"
+	dynasty=1051146
+	religion="nicene"
+	culture="greek"
+	trait="monk"
```

Comment:
A monk can not be played, did you intend to create a theocracy?

Reply:
Didn´t notice, just copied a character´s traits

Reply:
forgot to say "solved"